samples/bookinfo/src/productpage/productpage.py

```diff
@@ -15,6 +15,7 @@
 #   limitations under the License.
 
 
+from __future__ import print_function
```

Comment:
Does this import require any changes to the dockerfile for building the image?

Note that this changed productpage.py won't be used, unless we rebuild the image and push it to dockerhub. Maybe that's not really necessary, since the behavior isn't changed in this version.

Reply:
This import is part of the Python standard library and the only thing that it does is enforce Python 3 print() function syntax in Python 2 so that legacy print statements are a syntax error in both Py2 and Py3.